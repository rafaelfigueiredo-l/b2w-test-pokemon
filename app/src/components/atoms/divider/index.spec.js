import Divider from "./";

describe("<Divider />", () => {
  const component = <Divider color="red" width="100%" />;

  const mounted = mount(component);
  const wrapper = shallow(component);

  it("should mount correctly Divider component", () => {
    expect(wrapper.find(".divider")).toHaveLength(1);
    expect(mounted).toMatchSnapshot();
  });

  it("should unmount correctly Divider component", () => {
    expect(mounted.unmount()).toEqual({});
  });

  it("should render correctly color", () => {
    expect(wrapper.prop("color")).toEqual("red");
  });

  it("should render correctly width", () => {
    expect(wrapper.prop("width")).toEqual("100%");
  });
});
