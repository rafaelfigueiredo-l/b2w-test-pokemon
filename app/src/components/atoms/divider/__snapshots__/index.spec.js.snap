// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Divider /> should mount correctly Divider component 1`] = `
<styled.div
  className="divider"
  color="red"
  width="100%"
>
  <div
    className="sc-AxjAm eTaBtL divider"
    color="red"
    width="100%"
  />
</styled.div>
`;
